### DISCUSSION TITLE
Requirements for Concept #12 - Take-Save-Display Images

### DISCUSSION THREAD

**Discussion by jcasman** [caitlynbakery/theta_image_pick_11#5] [Ideas]
Requirements for new app!

Take a picture using the app

- Save the image to the Android image gallery
- Use image_picker to show the image
- Use gallery_saver to save the image
  - Previously, used list files, and saved the last file. This may not be necessary
- Use panorama to display the image in 360 with navigation

Optional
- Support for iPhone

**Comment by caitlynbakery** [comment]
Alright, I'll get started on it today!
